packages/mathtype-html-integration-devkit/src/contentmanager.js

```diff
@@ -447,6 +479,33 @@ export default class ContentManager {
     }
     this.updateToolbar();
     this.onFocus();
+
+    if (this.deviceProperties.isIOS) {
+      if (this.browserProperties.isSafari) {
+        const zoom = document.documentElement.clientWidth / window.innerWidth;
```

Comment:
Does this not work with browsers other than Safari? What's the behavior there?

Reply:
I checked that using an iPhone, and it is true. I removed the isSafari condition.